gulpfile.js

```diff
@@ -21,7 +21,7 @@ gulp.task('copy', () =>
 
 // Process the Stylus files into CSS
 gulp.task('styles', () =>
-  gulp.src('static/src/styles/**/*.styl')
+  gulp.src('static/src/styles/main.styl')
```

Comment:
Won't this only pick up changes to main and not recompile changes in other files?